.codeclimate.yml

```diff
@@ -0,0 +1,20 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
+
+  fixme:
+    enabled: true
+
+  rubocop:
+    enabled: true
+
+ratings:
+  paths:
+  - "**.rb"
+
+exclude_paths: 
+- lib/papertrail/okjson.rb
```

Comment:
I usually avoid adding codeclimate.yml (less clutter), but it is needed if we want to exclude the vendored `okjson` (which pulls the gpa down quite a bit...)